setup.py

```diff
@@ -40,18 +40,19 @@
               'pytest',
               'requests-mock',
               'vcrpy',
-              'rstcheck'
+              'rstcheck',
+              'sphinx'
```

Comment:
Napoleon requires sphinx >= 1.3 - think we should include this version dependencance.